src/sql/magic_cmd.py

```diff
@@ -65,8 +66,68 @@ def execute(self, line="", cell="", local_ns=None):
 
             args = parser.parse_args(others)
             return inspect.get_columns(name=args.table, schema=args.schema)
+        elif cmd_name == "test":
+            parser = CmdParser()
+
+            parser.add_argument(
+                "-t", "--table", type=str, help="Table name", required=True
+            )
+            parser.add_argument(
+                "-c", "--column", type=str, help="Column name", required=False
+            )
+            parser.add_argument(
+                "-w",
+                "--within",
+                type=str,
+                help="Whether it is within two numbers",
+                required=False,
+            )
+            parser.add_argument(
+                "-sg",
+                "--strictly-greater-than",
+                type=str,
+                help="Greater than a certain number.",
+                required=False,
+            )
+            parser.add_argument(
+                "-sl",
+                "--strictly-less-than",
+                type=str,
+                help="Less than a certain number.",
+                required=False,
+            )
+            parser.add_argument(
+                "-nn",
+                "--no-nulls",
+                help="Returns rows in specified column that are not null.",
+                action="store_false",
+            )
+
+            args = parser.parse_args(others)
+            query = construct_string_query(args)
+            conn = sql.connection.Connection.current.session
+            res = conn.execute(query).fetchall()
+            return res
         else:
             raise UsageError(
                 f"%sqlcmd has no command: {cmd_name!r}. "
                 "Valid commands are: 'tables', 'columns'"
             )
+
+
+def construct_string_query(args):
+    base_query = select("*").from_(args.table)
+    if args.within:
+        bottom, top = args.within.split(",")
+        where = condition(args.column + "<" + top).and_(args.column + ">" + bottom)
+        base_query = base_query.where(where)
+    if args.strictly_greater_than:
+        where = condition(args.column + ">" + args.strictly_greater_than)
+        base_query = base_query.where(where)
+    if args.strictly_less_than:
+        where = condition(args.column + "<" + args.strictly_less_than)
+        base_query = base_query.where(where)
+    if args.no_nulls:
```

Comment:
looks like you're filtering out NULLs but what we want is to raise an error if there's at least one NULL